src/ui/editortabwidget.cpp

```diff
@@ -197,6 +216,7 @@ void EditorTabWidget::tabRemoved(int)
 
 Editor *EditorTabWidget::currentEditor()
 {
+    setFocusProxy(editor(currentIndex()));
```

Comment:
Removing that line and adding a `setFocusProxy(editor);` to `EditorTabWidget::setFocus()` works, however.